proj/src/main/webapp/signup.js

```diff
@@ -171,7 +171,7 @@ function hasTag(){
 
 function enableSignUp(passVal){
     $('#tags').keyup(hasTag);
-    if (meetRequirements(passVal) && noMatch.hidden && hasTag()){
+    if (meetRequirements(passVal) && noMatch.hidden & hasTag()){
```

Comment:
Can you also add a comment about why we are using single & instead of &&?  Also can you either add parentheses around the first two or switch the first && to & also so that weird order of operations stuff doesn't happen where hasTags won't get called.

Reply:
sure thing